build.gradle.kts

```diff
@@ -185,8 +185,6 @@ if (!isRelease || isMainBranch) { // Only publish releases from the main branch
             channel.set(if (isRelease) "Release" else if (isMainBranch) "Snapshot" else "Alpha")
             changelog.set(changelogContent)
             apiKey.set(System.getenv("HANGAR_SECRET"))
-            owner.set("TheMeinerLP")
-            slug.set("BetterGoPaint")
```

Comment:
Shouldn't `id.set(“BetterGoPaint”)` be set there?

Reply:
No, the registration already acts as the ID